GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+Governance
+==========
+
+Overview
+--------
+sktime is a meritocratic, consensus-based community project. Anyone with an interest in the project can join the community, contribute to the project, and participate in the governance process. This document describes how that participation takes place and how to set about earning merit within the project community. We are particularly motivated to support new and/or anxious collaborators, people who are looking to learn and develop their skills, and anyone who has experienced discrimination in the past.
+
+In particular, we will outline
+
+* how we expect all members of the sktime community to behave,
+* what priviledges and responsibilities authors and maintainers have,
+* how we curate and select new contributions and existing functionality,
+* how we make decisions,
+* how we acknowledge contributions.
+
+
+Code of Conduct
+---------------
+We value the participation of every member of our community and want to ensure an that every contributor has an enjoyable and fulfilling experience. Accordingly, everyone who participates in the sktime project is expected to show respect and courtesy to other community members at all times.
+
+To make clear what is expected, we ask all members of the community to
+conform to our `Code of Conduct <https://github
+.com/alan-turing-institute/sktime/blob/master/CODE_OF_CONDUCT.rst>`_.
+
+Authorship and maintainership of algorithms
+-------------------------------------------
+
+Authors and maintainers of algorithms have special rights and duties with regard to changes and maintenance of their algorithms and associated documentation, including doc strings and tutorial notebooks.
+
+Algorithms are encapsulated in classes. To faciliate authorship and maintainership questions, each algorithm is written in a separate file.
+
+The author is the contributor who implements and contributes the algorithm to the project. The maintainer is the contributor who is responsible for maintaining an algorithm and the first point of contact for users and other contributors for all issues, questions, and proposals regarding their algorithm. The author automatically becomes the first maintainer of the algorithm.
+
+* **Veto right.** The maintainer has the right to veto any proposed changes to their algorithm within a time frame of 1 week. This does not extend to change to the common framework and API.
+
```

Comment:
what if they are on holiday?

Reply:
also, durations are typically in working days, not calendar days

Reply:
where one must note that different countries have different working days...

Reply:
I've made them quite short, because long wait times will likely discourage new contributors, but I'm happy to extend time frames, perhaps 10 working days, with working days Mon - Fri, ignoring public holidays

Reply:
Well, new contributors wouln't make changes to complete/established/curated algorithms as their first contribution? Even if, they should talk with the maintainer.

What about a two-stage process: 5 working days for first reaction, after which "suspected disengagement". Lack of reaction after 15 working days total = "assumed disengagment", then maintenance can be transferred.

If maintainer reappears later, this becomes a case for technical committee? 
Default consequence is split into two variant algorithms (in full library).

Reply:
ok, just read your text - that would double the 15 working days deadline. Also makes sense to me.